src/ring_theory/multiplicity.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2018 Robert Y. Lewis. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Robert Y. Lewis, Chris Hughes
 -/
-import data.nat.enat ring_theory.associated
+import algebra.gcd_domain data.nat.enat
```

Comment:
Does this use anything from `gcd_domain` or is a weaker import possible?

Reply:
done